components/modal/gallery/src/index.scss

```diff
@@ -56,15 +56,17 @@ $mt-modal-gallery-header-height-mobile: -($h-modal-basic-icon + ($m-l * 2)) !def
         justify-content: center;
       }
 
-      &-header {
-        margin-top: $mt-modal-gallery-header-height-mobile;
-      }
-
       &-content {
         flex-grow: 0;
       }
     }
 
+    @include media-breakpoint-down (xs) {
+      &-header {
+        margin-top: $mt-modal-gallery-header-height-mobile;
+      }
```

Comment:
i've been checking the demo, and i must say that i don't really understand why does we need a negative margin, but there must be some reason!
🔦could you bring some light?

Reply:
Header and content are both displayed vertically align. And following design specs we wanted the content to be perfectly centered so this negative margin does the trick by moving up everything a bit.